src/test/java/co/rsk/bitcoinj/script/ScriptTest.java

```diff
@@ -808,6 +808,261 @@ public void isSentToMultiSig_whenNonStandardErpRedeemScript_shouldReturnTrue() {
         assertScriptIsMultiSig(nonStandardErpRedeemScript);
     }
 
+    @Test
+    public void getSigsPrefixCount_whenP2PKScript_shouldReturnZero() {
+        BtcECKey signer = new BtcECKey();
+        Script p2pkScript = ScriptBuilder.createOutputScript(signer);
+        assertEquals(0, p2pkScript.getSigsPrefixCount());
+    }
+
+    @Test
+    public void getSigsSuffixCount_whenP2PKScript_shouldReturnZero() {
+        BtcECKey signer = new BtcECKey();
+        Script p2pkScript = ScriptBuilder.createOutputScript(signer);
+        assertEquals(0, p2pkScript.getSigsSuffixCount());
+    }
+
+    @Test
+    public void getSigsPrefixCount_whenMultiSigScript_shouldReturnOne() {
```

Comment:
For example here, multisig script might be an output script or a redeem script

Reply:
done